network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/profile_block_rounded.html

```diff
@@ -0,0 +1,30 @@
+{% extends "./base_streamfield_block.html" %}
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% block block_content %}
+	<div class="tw-row tw-py-4">
+		{% for block in self.profiles %}
+          {% with card=block.profile count=self.cards|length %}
+            {% image card.image fill-140x140 as img %}
+            <div class="tw-px-4 small:tw-w-full {% if count > 2 %} large:tw-w-1/3 medium:tw-w-1/3 {% else %} medium:tw-w-1/3 {% endif %} tw-mb-5">
```

Comment:
**Question** Can you add some context why we need the if statement? It seems like we apply `medium:tw-w-1/3` in anycase. Also, the width we set for the "large and above"breakpoint sets the same width as the "medium and above" breakpoint and should not be necessary.

Reply:
Again, these aren't needed - I thought they were, so I've removed them, thanks! 1bda82b11d9db204639d8875b7fb32d2bdf9aa54